crates/committed/src/checks.rs

```diff
@@ -319,13 +325,14 @@ pub(crate) fn check_autosquash(
     source: report::Source<'_>,
     message: &str,
     report: report::Report,
-) -> Result<bool, anyhow::Error> {
-    if AUTOSQUASH_PREFIXES.iter().any(|prefix| message.starts_with(prefix)) {
-        report(report::Message::error(source, report::Autosquash {}));
-        Ok(true)
-    } else {
-        Ok(false)
+) -> Result<(bool, Option<usize>), anyhow::Error> {
```

Comment:
Should we instead return the new message?  We could use `strip_prefix` rather than `starts_with`.